java/src/main/java/com/adhoc/flight/QueryRunner.java

```diff
@@ -75,6 +79,10 @@
                 description = "Enable encrypted connection")
         public boolean enableTls = false;
 
+        @Parameter(names = {"-dsv", "--disableServerVerification"},
+                description = "Enable disable server verification")
```

Comment:
Can we change the description here to just be "Disable TLS server verification."

Reply:
Make sensing it is related to TLS connection, fixed